rb/lib/selenium/webdriver/common/service.rb

```diff
@@ -36,17 +36,31 @@ class Service
       SOCKET_LOCK_TIMEOUT = 45
       STOP_TIMEOUT        = 20
 
+      @executable = nil
+      @missing = ''
+
+      class << self
+        attr_accessor :executable, :missing_text
```

Comment:
It doesn't hurt anything, but it really only needs to be `attr_reader` since it is never set (and isn't intended to be) in the code. Updated, thanks again